src/adapters/sqlite/encodeSchema/index.js

```diff
@@ -65,6 +170,10 @@ const encodeAddColumnsMigrationStep: (AddColumnsMigrationStep) => SQL = ({
       )} = ${encodeValue(nullValue(column))};`
       const addIndex = encodeIndex(column, table)
 
+      if (column.isFTS) {
+        logger.warn('[DB][Worker] Support for migrations and isFTS is still to be implemented')
```

Comment:
oof, that's pretty bad

Reply:
err... https://github.com/Nozbe/WatermelonDB/pull/623#discussion_r382170393